setup.py

```diff
@@ -27,6 +27,9 @@
     python_requires=">=3.7",
     install_requires=install_requires,
     extras_require={"dev": dev_requires},
+    entry_points={
+        "prefect.collections": ["DbtCloudCredentials = prefect_dbt.cloud.credentials"]
```

Comment:
Fancy

Reply:
This is the piece that allows the `DbtCloudCredentials` block to be discovered by the core library and shown in the UI.